src/test/aztec/base/BridgeTestBase.sol

```diff
@@ -337,178 +349,178 @@ abstract contract BridgeTestBase is Test {
+            mstore(add(data, 0xe40), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xe60), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xe80), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xea0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xec0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xee0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xf00), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xf20), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xf40), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xf60), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xf80), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xfa0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xfc0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0xfe0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1000), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1020), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1040), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1060), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1080), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x10a0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x10c0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x10e0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1100), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1120), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1140), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1160), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore(add(data, 0x1180), 0x14e0f351ade4ba10438e9b15f66ab2e6389eea5ae870d6e8b2df1418b2e6fd5b)
+            mstore(add(data, 0x11a0), sload(rollupBeneficiary.slot))
```

Comment:
Please use the same structure as the other storage values. e.g., put them up top so it is easy to see what is changed.

Reply:
Addressed in a3eb8bce7dd21f52b8de828256e5d19ca167d0a8